client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { createUseStyles, useTheme } from "react-jss";
+
+const useStyles = createUseStyles({
+  pointsNotification: {
+    border: "2px solid transparent",
+    backgroundColor: "#a7c539",
+    color: "#0F2940",
+    boxShadow: "0px 0px 5px rgba(0, 0, 0, .2)",
+    marginTop: "16px",
+    maxWidth: "250px",
+    display: "flex",
+    justifyContent: "center",
+    cursor: "pointer",
+    zIndex: "1",
+    padding: "16px 24px",
+    lineHeight: "1.4",
+    margin: "auto"
+  }
+});
+
+const EarnedPointsMetContainer = () => {
+  const theme = useTheme();
+
+  const classes = useStyles({ theme });
```

Comment:
You passed theme in here but didn't actually use it in the createUseStyles

Reply:
Using theme now for theme colors